{"name": "test_login[req_body2-exp_data2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '用户名或密码错误！' == '账号不存在！'\n  - 账号不存在！\n  + 用户名或密码错误！", "trace": "self = <testcase.test_login.TestLogin object at 0x11066aa10>\nreq_body = {'password': '123456789', 'username': 'test'}\nexp_data = {'code': 401, 'msg': '账号不存在！'}\n\n    @pytest.mark.parametrize('req_body,exp_data',get_yaml_data('../data/loginCase.yaml'))\n    def test_login(self, req_body, exp_data):\n        #1.调用登录接口\n        #2.Login加个括号代表实例化\n        #3.数据驱动\n        res=Login().login(req_body)\n        #print(res.json().get('msg'))\n>       assert res.json().get('msg') == exp_data['msg']\nE       AssertionError: assert '用户名或密码错误！' == '账号不存在！'\nE         - 账号不存在！\nE         + 用户名或密码错误！\n\ntest_login.py:19: AssertionError"}, "parameters": [{"name": "req_body", "value": "{'username': 'test', 'password': '123456789'}"}, {"name": "exp_data", "value": "{'code': 401, 'msg': '账号不存在！'}"}], "start": 1695118203552, "stop": 1695118203626, "uuid": "c4368ea1-fec4-42cc-8fab-fb227c65b2fb", "historyId": "51a2229a340f29086d4218c4a68b1cfd", "testCaseId": "c54ba4c514553f220eeed2f7e798b554", "fullName": "testcase.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HuimindeMacBook-Air.local"}, {"name": "thread", "value": "27839-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}